x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/bulk_actions/route.ts

```diff
@@ -131,12 +146,38 @@ const buildBulkResponse = (
     deleted?: RuleAlertType[];
   }
 ) => {
+  // Split errors into expected skipped rules and other errors
+  const { skipped, errors: errorsWithoutSkipped } = errors.reduce<ErrorsAndSkippedRules>(
+    (acc: ErrorsAndSkippedRules, error: BulkActionError) => {
+      if (
+        isBulkEditError(error) &&
+        Object.values(BulkEditSkipReason).includes(error.message as BulkEditSkipReason)
+      ) {
+        return {
+          ...acc,
+          skipped: [
+            ...acc.skipped,
+            { ...error.rule, bulk_edit_skip_reason: error.message as BulkEditSkipReason },
```

Comment:
`...error.rule`: Again, not sure about returning whole rule objects here, I'd return a subset of attributes:

- `id`: object id of the rule (`rule.id`) - always, because it is always known in the context of bulk actions
- `name` - optional, if known